yt_dlp/extractor/rutube.py

```diff
@@ -16,12 +16,16 @@
 
 
 class RutubeBaseIE(InfoExtractor):
-    def _download_api_info(self, video_id, query=None):
+    def _download_api_info(self, video_id, query=None, secret=None):
```

Comment:
secret should just be passed as part of `query`. Same below

Reply:
Yep, thanks!